core/src/main/java/org/apache/iceberg/util/SnapshotUtil.java

```diff
@@ -22,13 +22,21 @@
 import java.util.List;
 import java.util.function.Function;
 import org.apache.iceberg.DataFile;
+import org.apache.iceberg.HistoryEntry;
+import org.apache.iceberg.Schema;
 import org.apache.iceberg.Snapshot;
 import org.apache.iceberg.Table;
 import org.apache.iceberg.exceptions.ValidationException;
+import org.apache.iceberg.relocated.com.google.common.base.Preconditions;
 import org.apache.iceberg.relocated.com.google.common.collect.Iterables;
 import org.apache.iceberg.relocated.com.google.common.collect.Lists;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class SnapshotUtil {
```

Comment:
I don't see the static methods added here, `snapshotIdFromTime` and `schemaOfSnapshot`, used anywhere in this change. If that is the case, this file can be excluded from this particular PR.

Reply:
sounds good, will remove and create a separate PR for that.